src/quic/node_quic_socket-inl.h

```diff
@@ -79,8 +79,8 @@ void QuicSocket::AssociateStatelessResetToken(
   token_map_[token] = session;
 }
 
-const SocketAddress& QuicSocket::local_address() {
-  CHECK(preferred_endpoint_);
+const SocketAddress QuicSocket::local_address() const {
```

Comment:
`const` on non-referende/pointer return types does nothing, but leads to a compiler warnings sometimes